solc/CommandLineInterface.cpp

```diff
@@ -420,7 +420,16 @@ void CommandLineInterface::readInputFilesAndConfigureRemappings()
 bool CommandLineInterface::parseLibraryOption(string const& _input)
 {
 	namespace fs = boost::filesystem;
-	string data = fs::is_regular_file(_input) ? contentsString(_input) : _input;
+	string data = _input;
+	try
+	{
+		if (fs::is_regular_file(_input))
```

Comment:
Yes, by creating such a file, you change the behaviour of the compiler. In general, creating or modifying files whose names are mentioned on the commandline changes the behaviour of the compiler.